Change file permissions to read/write/execute for everyone.

jamarw opened the pull request:
Change file permissions to read/write/execute for everyone using chmod 777.  Resolves preview issue below:
![screen shot 2018-03-07 at 2 31 09 pm](https://user-images.githubusercontent.com/11825888/37123322-a3c7f250-2218-11e8-9a91-769540e0217f.png)

adkinn commented:
@herohua the new VS Code extension is in security review and will be made publicly available as soon as possible.  Could we get this tiny fix reviewed in order to improve the Preview experience on Mac?

jamarw commented:
Hi @vwxyzh , @superyyrrzz and @vicancy,

The PR could not be merged due to file changes (rename, move) in common.ts so I updated my fork and created a new clean PR.  I will close this PR now.  A link to the new PR is below.  I also added a comment and mentioned you in PR 2502.

Thank you and sorry for the churn.

https://github.com/dotnet/docfx/pull/2502
